tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -171,18 +189,20 @@ def optimal(
       if final node has more than one edge,
       `output_edge_order` must be provided.
     memory_limit: Maximum number of elements in an array during contractions.
+    ignore_edge_order: An option to ignore the output edge order.
 
   Returns:
     The network after full contraction.
   """
   alg = functools.partial(opt_einsum.paths.optimal, memory_limit=memory_limit)
-  return base(nodes, alg, output_edge_order)
+  return base(nodes, alg, output_edge_order, ignore_edge_order)
 
 
 def branch(nodes: Union[TensorNetwork, Iterable[BaseNode]],
-           output_edge_order: Optional[Sequence[Edge]] = None,
+           output_edge_order: Sequence[Edge] = None,
```

Comment:
Don't remove the `Optional` keyword here.

Reply:
Sorry think this is just where my merge went wonky.